plans/all.fmf

```diff
@@ -1,38 +1,99 @@
 summary: top level management
 
+enabled: true
 adjust:
   when: plan == cockpit
   enabled: false
 
-prepare:
-  - name: Start TPM2 emulation kernel module
-    how: shell
-    script: modprobe tpm_vtpm_proxy
-  - name: Install packages
-    how: install
-    package:
-     - tang
-     - systemd
-     - swtpm
-     - swtpm-tools
-     - tpm2-tools
-  - name: Start TPM2 emulation
-    how: shell
-    script: mkdir /var/tmp/swtpm; swtpm_setup --tpm-state /var/tmp/swtpm --tpm2; swtpm chardev --vtpm-proxy --tpmstate dir=/var/tmp/swtpm --tpm2 &> /var/log/swtpm &
-  - name: Start tang server
-    how: shell
-    script: systemctl enable tangd.socket --now
-  - name: Reload udev
-    how: shell
-    script: udevadm control --reload
-  - name: Show test system information
-    how: shell
-    script: free -m; lsblk -i; lscpu; cat /proc/1/sched
-  - name: Record mkfs.xfs version
-    how: shell
-    script: mkfs.xfs -V
-discover:
-  how: fmf
-execute:
-  how: tmt
-  exit-first: false
+/python:
```

Comment:
You can use inheritance here, and should use it to make it clear what is different between the two setups. It turns out that the inheritance works fine, so something like this will do: https://github.com/stratis-storage/stratisd/pull/3640/commits/690e2066c9480dce8aaca120e6b2af38b4dfe891

Reply:
Ah, I think I missed the part on inheritance and the factored out step was just being replaced by the specified step.

Reply:
Resolved.